style.css

```diff
@@ -0,0 +1,119 @@
+body {
+  width: 375px;
+  height: 718px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  cursor: pointer;
+  z-index: +1;
+}
+
+.logo-image {
+  height: auto;
+  width: 50%;
+  margin-left: 10%;
+  margin-right: 10%;
+}
+
+.bar {
+  display: flex;
+  cursor: pointer;
+  flex-direction: column;
+  width: 35px;
+  height: 5px;
+  background-color: black;
+  margin: 3px 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./public/headershapes-mobile.svg);
+  position: absolute;
+  align-items: center;
+  width: 375px;
+  height: 768px;
+  left: 0%;
+  top: 0%;
+}
```

Comment:
- [ ]  I have noticed an issue with the responsiveness of your webpage on smaller devices. The page appears to overflow and does not adjust properly to mobile screen sizes.
- Make sure your images are also responsive by setting their max-width to 100% in CSS. This ensures that images automatically scale within their parent containers on smaller screens.
- Review your HTML and CSS code to ensure that these recommendations are implemented properly. Test your webpage on various mobile devices and screen sizes to ensure it responds appropriately and does not overflow.

- [ ] Your design
![image](https://github.com/Kaiserabbas/Portfolio/assets/68823331/e314b65e-e07b-4826-b9d4-efa6c7b97015)